### PR TITLE
Fix NMR print

### PR DESCRIPTION
**Pull request by stigrj**

### CODE REVIEW

File: doc/users/user_inp.rst [hunk 1]
```diff
@@ -299,7 +299,10 @@ The ``nuclear_specific`` keyword triggers response calculations using the
 nuclear magnetic moment operator instead of the external magnetic field. For
 small molecules this is not recommended since it requires a separate response
 calculation for each nucleus, but it might be beneficial for larger systems if
-you are interested only in a single shielding constant.
+you are interested only in a single shielding constant. Note that the components
+of the *perturbing* operator defines the *row* index in the output tensor, so
+``nuclear_specific = true`` will result in a shielding tensor which is
+the transpose of ``false``.
```
Comment (robertodr) [inline doc/users/user_inp.rst:305]: "the transpose of the one obtained by setting `nuclear_specific = false`". More verbose, but less ambiguous. I think.

File: src/properties/NMRShielding.h [hunk 2]
```diff
@@ -53,12 +53,12 @@ class NMRShielding final {
 
     void print(const std::string &id) const {
         auto sigma = getTensor();
-        Eigen::SelfAdjointEigenSolver<DoubleMatrix> es;
+        Eigen::EigenSolver<DoubleMatrix> es;
```
Comment (robertodr) [inline src/properties/NMRShielding.h:56]: Why is the matrix no longer self-adjoint?

File: src/qmoperators/one_electron/H_MB_dia.h [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "NuclearGradientOperator.h"
+#include "PositionOperator.h"
+#include "qmoperators/RankTwoTensorOperator.h"
+
+namespace mrchem {
+
```
Comment (robertodr) [inline src/qmoperators/one_electron/H_MB_dia.h:8]: Can you add a Doxygen string explaining what this operator is? Maybe also for the `H_BM_dia` one, while you're at it :grin:

**Review by robertodr — APPROVED** [review]

File: src/properties/NMRShielding.h (hunk 2 above)
Comment (stigrj) [reply src/properties/NMRShielding.h:56]: It shouldn't be. It was an error before which lead to wrong calculation of anisotropy. I guess it was only tested on symmetric cases?

**Review by robertodr — APPROVED** [review]
So much documentation added, I'm awed :smile:

File: src/properties/NMRShielding.h (hunk 2 above)
Comment (robertodr) [reply src/properties/NMRShielding.h:56]: And what happens with the imaginary part of the eigenvalues?

Comment (stigrj) [reply src/properties/NMRShielding.h:56]: No idea, but now we're consistent with orca :slightly_smiling_face:  It might never become complex?

Comment (stigrj) [reply src/properties/NMRShielding.h:56]: I'm adding a warning in case that happens

**Pull request merged**